index.bs

```diff
@@ -265,15 +264,13 @@ attribute</a> for the {{change}} event type.
 The <dfn method for=IdleDetector>requestPermission()</dfn> method steps are:
 
 1.  If the [=relevant global object=] of [=this=] does not have [=transient
-    activation=], return [=a new promise=] [=rejected=] with a "{{NotAllowedError}}"
+    activation=], return [=a promise rejected with=] a "{{NotAllowedError}}"
     {{DOMException}}.
 1.  Let |result| be [=a new promise=].
 1.  [=In parallel=]:
     1.  Let |permissionState| be the result of [=requesting permission to use=]
-        the [=powerful feature=] named `"idle-detection"`.
-    1.  [=Queue a global task=] on the [=relevant global object=] of [=this=]
-        using the [=permission task source=] to [=resolve=] |result| with
-        |permissionState|.
+        `"idle-detection"`.
+    1.  [=Resolve=] |result| with |permissionState|.
```

Comment:
Since we're running in parallel we do need to queue a task to resolve here. Though... since resolving a promise itself queues a microtask maybe it's not actually necessary? I guess we still need to know which task source executes that microtask.

Reply:
> since resolving a promise itself queues a microtask maybe it's not actually necessary?

That's been my understanding.
 
>  I guess we still need to know which task source executes that microtask.

I'm not sure. My understanding is that you just resolve with the value and the micro task does the right thing without the needing to involve any additional task source (i.e., go into parallel, get the value you need over IPC or whatever, then just call resolve). 

We can check with Domenic or someone that knows these bits better tho?

Reply:
Haven't look at the full context, but no, you need to be on a queued task to use "resolve". We used to have it auto-queue a task but we stopped that a while ago; sorry for the confusion.

Reply:
Thanks for confirming @domenic!